packages/modular-scripts/src/__tests__/index.test.ts

```diff
@@ -287,17 +285,23 @@ describe('modular-scripts', () => {
       },
     });
 
-    // I think this is wrong, it's not running tests
-    // in sample-package and sample-view
-    expect(censorVariableTimes(output.all)).toMatchInlineSnapshot(`
+    // TODO: Passing CI=true *should* remove all the coloring stuff,
+    // it's weird that it doesn't. To workaround it, I've manually
+    // removed those tokens from the string for the snapshot test.
+    // Open to suggestions/fixes.
+
+    // eslint-disable-next-line no-control-regex
+    expect(censorVariableTimes(output.all?.replace(/|\[\d+./gm, '')))
+      .toMatchInlineSnapshot(`
       "$ ts-node packages/modular-scripts/src/cli.ts test sample-app sample-package sample-view
-      PASS ../sample-app/src/__tests__/App.test.tsx
-        ✓ renders learn react link (? ms)
+       PASS  ../sample-package/src/__tests__/index.test.ts
```

Comment:
The tests are failing because the order of these 3 tests is not deterministic. I'm not even sure how to begin to solve that, so I'm leaving this as is for now while I think about it. Everything else seems a-ok tho.